services/director-v2/src/simcore_service_director_v2/models/dynamic_services_scheduler.py

```diff
@@ -427,6 +428,14 @@ def endpoint(self) -> AnyHttpUrl:
         default=None,
         description="contains information about the wallet used to bill the running service",
     )
+    pricing_info: PricingInfo | None = Field(
```

Comment:
ok, since this is already optional. why make all its fields as well optional? I don't really see the advantage of the object then

Reply:
yes as mentioned above my mistake i didn't want to make them all optional, I will fix it.